python/lsst/ts/ess/csc/ess_csc.py

```diff
@@ -35,8 +35,11 @@
 from lsst.ts import salobj, tcpip
 from lsst.ts.ess import common
 
-"""Standard timeout in seconds for socket connections."""
 SOCKET_TIMEOUT = 5
+"""Standard timeout in seconds for socket connections."""
+
+NUMBER_OF_TEMP_CHANNELS = 16
```

Comment:
Please spell out temperature. I initially read "temp" as short for "temporary"

Reply:
Done